pkg/db/migration/convert/convert000033/convert.go

```diff
@@ -0,0 +1,195 @@
+package convert000033
+
+import (
+	"fmt"
+	"strings"
+
+	"go.uber.org/zap"
+	"gopkg.in/yaml.v3"
+	"gorm.io/gorm"
+
+	"github.com/instill-ai/pipeline-backend/pkg/datamodel"
+	"github.com/instill-ai/pipeline-backend/pkg/db/migration/convert"
+)
+
+const batchSize = 100
+
+type ConvertWebFields struct {
+	convert.Basic
+}
+
+var webCompNames []string
+
+func (c *ConvertWebFields) Migrate() error {
+	if err := c.migratePipeline(); err != nil {
+		return err
+	}
+
+	if err := c.migratePipelineRelease(); err != nil {
+		return err
+	}
+
+	if len(webCompNames) > 0 {
+		return fmt.Errorf("web components' output are not updated: %v", webCompNames)
```

Comment:
Is this an error, though? Ideally the migration should be idempotent, if you run it twice it would change no components and it wouldn't be an error.